documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -137,3 +144,10 @@ Read our troubleshooting steps, tips, and tricks here: xref:display.adoc#trouble
 * Outer dimensions: 192.96 × 110.76mm
 * Viewable area: 154.08 × 85.92mm
 * https://datasheets.raspberrypi.com/display/7-inch-display-mechanical-drawing.pdf[Download mechanical drawing (PDF)]
+
+[.whitepaper, title="Touch Display Enclosure Mechanical Design", subtitle="", link=https://pip.raspberrypi.com/categories/685-whitepapers-app-notes/documents/RP-003473-WP/Touch-Display-Enclosure-Mechanical-Design.pdf]
+****
+This whitepaper provides information for designing mechanical enclosures for the Raspberry Pi Touch Display.
+
+The Raspberry Pi Touch Display is dual-source manufactured, and there are very slight differences in the mechanical design between the two manufacturers. This document describes those differences and provides possible mitigation mechanisms for ensuring that enclosure designs can incorporate either version of the panel. For the purposes of this document, the two versions of the panel will be designated 'Supplier A' and 'Supplier B'. Raspberry Pi Ltd will not be publishing the names of the manufacturers used for the liquid crystal display (LCD) panel, or the panel datasheets.
```

Comment:
I'm not sure if this is "too much detail" to include in the documentation itself, and whether a briefer summary would be more suitable?

Reply:
Agreed. Summary needs to be shorter. But it definitely should be elsewhere.